src/aap_eda/core/models/credential_type.py

```diff
@@ -34,5 +36,14 @@ class Meta:
     managed = models.BooleanField(default=False)
     kind = models.TextField(default="cloud", blank=True, null=False)
     namespace = models.TextField(default=None, null=True)
+    organization = models.ForeignKey(
+        "Organization", on_delete=models.CASCADE, null=True
```

Comment:
@Dostonbek1 Can we use a default field here akin to this https://github.com/ansible/eda-server/blob/d99a07a23852bae28f896c4ccb2f17292bd37d84/src/aap_eda/core/models/event_stream.py#L89 and we can skip all the save calls.